bot/ctf_model.py

```diff
@@ -294,9 +294,9 @@ async def add_chal(self, name):
         chan = await catg_working.create_text_channel(
             name=fullname, overwrites=overwrites
         )
-        Challenge.create(guild, cid, chan.id, name)
+        Challenge.create(guild, channel_id, chan.id, name)
         chk_upd(
-            fullname, teams.update_one({"chan_id": cid}, {"$push": {"chals": chan.id}})
+            fullname, teams.update_one({"channel_id": channel_id}, {"$push": {"chals": chan.id}})
```

Comment:
These database operations (`update_one`, `find_one`, etc.) might in fact be an issue as this would break old CTFs before this change.

I would say we should keep the key names (`chan_id`, etc.) for now and rather only change the Python variable names.

A solution to this would be to implement a legacy checker that would search for objects that match for example `chan_id` or `channel_id`, that should solve that issue.